src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -929,6 +941,14 @@ public function __call($method, $parameters)
 		{
 			return $this->callScope($scope, $parameters);
 		}
+		elseif ( ! is_null($this->model['fillable']) && in_array($column = 'is_'.$method, $this->model['fillable']))
```

Comment:
And even faster if you would store `$this->model['fillable']` in a local variable